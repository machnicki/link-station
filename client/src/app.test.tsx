import App from "./app";
import { render } from "@testing-library/react";

describe("#App", () => {
  it("should have proper structure", () => {
    const { container } = render(<App />);
    expect(container).toMatchSnapshot();
  });
});
