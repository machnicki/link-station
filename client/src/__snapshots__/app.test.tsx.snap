// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#App should have proper structure 1`] = `
<div>
  <div
    class="App"
  >
    <h1>
      Find the best link station
    </h1>
    <div
      class="container"
    >
      <div
        class="row"
      >
        <div
          class="col"
        >
          <div>
            <div
              class="list-group"
            >
              <button
                class="list-group-item list-group-item-action"
              >
                Device with coordinates X: 0, Y: 0
              </button>
              <button
                class="list-group-item list-group-item-action"
              >
                Device with coordinates X: 100, Y: 100
              </button>
              <button
                class="list-group-item list-group-item-action"
              >
                Device with coordinates X: 15, Y: 10
              </button>
              <button
                class="list-group-item list-group-item-action"
              >
                Device with coordinates X: 18, Y: 18
              </button>
            </div>
          </div>
        </div>
        <div
          class="col"
        >
          <div
            class="Results"
          >
            <h2>
              Choose device
            </h2>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
